tasks/setup_owncloud.yml

```diff
@@ -128,31 +128,6 @@
   ## Did not work with 8.1.6-2.1 because of https://github.com/owncloud/core/issues/17583
   ## Falling back to URL auto install method.
 
-- name: Install ownCloud autoconfig file
-  template:
-    src: 'srv/www/sites/config/autoconfig.php.j2'
-    dest: '{{ owncloud__deploy_path }}/config/autoconfig.php'
-    owner: '{{ owncloud__user }}'
-    group: '{{ owncloud__group }}'
-    mode: '0660'
-    validate: 'php -f %s'
-  register: owncloud__register_autoconfig
-  when: owncloud__do_autosetup|bool and owncloud__release in [ '8.0' ]
-
-- name: Test availability of automatically finish setup
-  uri:
-    url: '{{ owncloud__autosetup_url }}'
-    status_code: '200,401'
-  register: owncloud__auto_finish_setup
-  when: not owncloud__register_autoconfig|skipped
-
-- name: Automatically finish setup via URL request
```

Comment:
Setup is still automated but using the URL get method for all supported releases.

> I'm guessing that this is part of the code that's removed due to 8.2 support being dropped?

Yes, the 3 tasks where only run for `owncloud__release in [ '8.0' ]`.